### ISSUE TITLE
Does score_converter should be SOFTMAX?

### ISSUE THREAD

**Issue by emitoooo** [bosch-ros-pkg/bstld#11]
Hi, you have multiple classes then your score_converter should be SOFTMAX? Why did you use SIGMOID?

**Comment by karstenBehrendt** [comment]
Hi @etatbak - Can you please point me to that specific line or do you mean in the paper?

**Comment by emitoooo** [comment]
Thanks for you reply. I meant in config file:
`score_converter : SIGMOID`

**Comment by karstenBehrendt** [comment]
Good question.

I may have just copied some config from the object detection repo, e.g. https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/ssd_mobilenet_v2_coco.config and have not looked into it. This coco config uses sigmoids.
Maybe, I copied class independent object detectors. That was some time ago.

I will not be able to evaluate changes within the next two weeks, so it will be a while until I can look into this.

**Comment by emitoooo** [comment]
Yes I see but the original config file of faster_rcnn_nas https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/faster_rcnn_nas_coco.config   is  `score_converter: SOFTMAX`

**Comment by karstenBehrendt** [comment]
As I said, it will likely perform a tad bit better using softmax but I currently do not have the time to look into it. Feel free to fork the repo, train a model, and report the updated accuracy on the test set.

Since this is more of a performance update than a bug, I will close the ticket. Feel free to submit changes
